HourlyERA5_to_DailyERA5.py

```diff
@@ -0,0 +1,116 @@
+# -*- coding: utf-8 -*-

```

Comment:
@AkaMrmuffin Mo - Can you please add the license heading (like other files) and a description of what this file does at the top?

Reply:
Updated